zilencer/management/commands/populate_db.py

```diff
@@ -622,7 +622,7 @@ def handle(self, **options: Any) -> None:
                     extra_stream_name = 'Extra Stream ' + number_str
 
                     zulip_stream_dict[extra_stream_name] = {
-                        "description": "Auto-generated extra stream.",
+                        "description": "Auto-generated extra stream for *testing* `code`.",
```

Comment:
Can we have a function which returns stream description that randomly includes these just for the extra streams:
* links
* emoji
* some markdown text

Reply:
How many different links, emoji, and markdown syntax choices should I have for those extra streams?